tools/packaging/packaging.strings.psd1

```diff
@@ -148,7 +155,30 @@ RefAssemblyCsProj = @'
   </ItemGroup>
 </Project>
 '@
-NuGetConfigFile = @'
+
+    'Microsoft.PowerShell.Commands.Utility' = @'
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Version>{0}</Version>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>{1}</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <LangVersion>Latest</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System.Management.Automation">
+    <HintPath>{2}</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
```

Comment:
How do we manage updating versions of these references when an update is published for them?
Can you open an issue to track this work?

Reply:
Yes, issue opened: #8721